# Amplicon analysis in R

## Resources
- [Intro to Amplicon Sequencing Presentation (PDF)](Lecture_Intro_Amplicon_Sequencing_Day3_Crandall.pdf)
- [Download data files and code for 16S R tutorial](https://drive.google.com/file/d/108D4IyIGRfav6NTDiwkpOP-CpNBQgWMW/view?usp=sharing)
- [Intro to Microbiome Viz Presentation (PDF)](MicrobiomeViz.pdf)


# **Suggested Reading**

Weinroth MD, Belk AD, Dean C, et al. Considerations and best practices in animal science 16S ribosomal RNA gene sequencing microbiome studies. J Anim Sci. 2022;100(2):skab346. doi:10.1093/jas/skab346 [Link](https://academic.oup.com/jas/article/100/2/skab346/6519592)

Ginnan N, Bordenstein SR. It is time to authenticate the Microbiome Sciences with accredited educational programs and departments. PLoS Biol. 2023 Dec 7;21(12):e3002420. doi: 10.1371/journal.pbio.3002420. PMID: 38060452; PMCID: PMC10703218. [Link](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002420)

Mirzayi C, Renson A; Genomic Standards Consortium; Reporting guidelines for human microbiome research: the STORMS checklist Nat Med. 2021 Nov;27(11):1885-1892. doi: 10.1038/s41591-021-01552-x. Epub 2021 Nov 17. PMID: 34789871; PMCID: PMC9105086.[Link](https://www.nature.com/articles/s41591-021-01552-x)

Callahan BJ, McMurdie PJ, Rosen MJ, Han AW, Johnson AJ, Holmes SP. DADA2: High-resolution sample inference from Illumina amplicon data. Nat Methods. 2016;13(7):581-583. doi:10.1038/nmeth.3869 [Link](https://www.nature.com/articles/nmeth.3869)

Gloor GB, Macklaim JM, Pawlowsky-Glahn V, Egozcue JJ. Microbiome Datasets Are Compositional: And This Is Not Optional. Front Microbiol. 2017;8:2224. Published 2017 Nov 15. doi:10.3389/fmicb.2017.02224 [Link](https://www.frontiersin.org/journals/microbiology/articles/10.3389/fmicb.2017.02224/full)

# **Required Software**

Software:
[R and Rstudio](https://cran.rstudio.com/)

Packages: 
[dada2](https://bioconductor.org/packages/release/bioc/html/dada2.html) , [phyloseq](https://bioconductor.org/packages/release/bioc/html/phyloseq.html), and [tidyverse](https://tidyverse.tidyverse.org/)

# **Tutorial Files from Microbiome Center Workshop**
[Link to 16s amplicon section of 2023 workshop](https://github.com/Penn-State-Microbiome-Center/KickStart-Workshop-2023/tree/main/Day2-AmpliconR)

# **Short lectures**
[Link to an overview of sequencing methodologies](https://psu.mediaspace.kaltura.com/media/Next+Generation+Sequencing/1_12vugl0d)

[Link to an overview of bioinformatics](https://psu.mediaspace.kaltura.com/media/Overview+of+Bioinformatics+and+Diversity+Metrics/1_1ih13k9f)

# **Suggested Resources**

## Background: Coursera - gut check
It is great to start grappling with concepts - videos are short<br>
https://www.coursera.org/lecture/microbiome/welcome-to-the-course-AuMZk

## Coding:

#### üñ•Ô∏è Getting started with R
1.	https://www.codecademy.com/learn/learn-r
2.	https://www.codecademy.com/search?query=R
   
#### üíª Getting started with BASH
Before you do biostars handbook do this quick shell one
https://swcarpentry.github.io/shell-novice/

#### üßë‚Äçüíª The best way to get started with bioinformatics
For getting used to best practices in coding in the command line<br>
https://www.biostarhandbook.com/index.html

#### ‚úÖ Scientific rigor, reproducibility, version control, and code-alongs
Riffomonas has great tutorials to get started with R and reproducible research with version control is also a must.
https://riffomonas.org/

#### üß¨ 16s analysis
**16S analysis in R**<br>
https://benjjneb.github.io/dada2/tutorial.html<br>

**16S analysis in Qiime2**<br>
https://docs.qiime2.org/2021.11/tutorials/overview/
